lib/components/atoms/FileInput/FileInput.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React from 'react';
+
+import styled from 'styled-components';
+import styles from './FileInput.style';
+
+type Props = {
```

Comment:
Move this to specific types file and export it from there so that here and in any other component types can be reused

Reply:
![image](https://user-images.githubusercontent.com/49902237/70633408-89082900-1c56-11ea-9f1c-68a7ac535393.png)

I will discuss further on this.